types.go

```diff
@@ -223,7 +223,15 @@ type User struct {
 }
 
 type ProfilePicture struct {
-	Urls []string
+	Urls         []string
```

Comment:
I have been thinking about what you are doing here. I like the idea about it.  However it is diverging quite a bit from the original type and away from the tiktok protobuf type too.

The original code was very opinionated about what picture to return, and now with this change it would be opinionated about what is the best an these other URLs (shrink, default, jpeg, etc) on top of the "best" return is odd.  I guess tiktok is still sending various urls even for AvatarLarge or AvatarJpeg?

Let me know if you want to still work on this, but I want to take this in a different direction. I want to remove user.ProfilePicture completely, keep the ProfilePicture type, but add the fields tiktok has to user instead:
AvatarLarge, AvatarMedium, AvatarThumb, and AvatarJpeg.

This does two things:  1. Make the return types from this library closer to what tiktok provides and other libraries. 2. Leave the opinion about what is "best" to the library user.  So you would still use some kind of sorting in your library, if that's what you want, to choose, but let's not make that choice and algorithm part of this library itself.